lang/en/permission.php

```diff
@@ -36,6 +36,7 @@
         Role::FOUNDER => __('Founder'),
         Role::ARCHITECT => __('Architect'),
         Role::ENGINEER => __('Engineer'),
+        Role::POINT_OF_CONTACT => __('Team Account'),
```

Comment:
If "Team Account" is a better name for the role, now would be the time to change it. Once it's in the DB, it's a lot harder.

Reply:
Agreed - done.